cost/flexera/cco/scheduled_reports/scheduled_report.pt

```diff
@@ -543,13 +602,96 @@ script "js_collated_data", type: "javascript" do
       month = timestamp.toISOString().split('-')[1]
       stringMonth = ds_data_tables['months_short'][Number(timestamp.toISOString().split('-')[1])]
       stringDay = stringMonth + "-" + day
+      // Get days in month for yearMonth
+      var daysinTimeUnit = new Date(year, month, 0).getDate()
 
       category_id = ds_graph_dimension['id']
       if (category_id == "billing_center_id") { category_id = "billing_center_name" }
       category = row['dimensions'][category_id]
 
-      if (param_billing_term == 'Day')   { timeUnit = timestamp.toISOString().split('T')[0] }
-      if (param_billing_term == 'Month') { timeUnit = [year, month].join('-') }
+      if (param_billing_term == 'Day')   {
+        timeUnit = timestamp.toISOString().split('T')[0]
+        // Override daysinTimeUnit for daily average calculation
+        // At param_billing_term Day, this Average is not useful and will match the cost
+        daysinTimeUnit = 1
+      }
```

Comment:
Extremely minor feedback but it might make sense to have at least one empty line between { } code blocks for readability